common/components/AnkiDialog.tsx

```diff
@@ -352,7 +352,7 @@ const AnkiDialog = ({
     ]);
 
     const initialAudioClip = useMemo(
-        () => AudioClip.fromCard(card, settings.audioPaddingStart, settings.audioPaddingEnd, false),
+        () => AudioClip.fromCard(card, settings.audioPaddingStart, settings.audioPaddingEnd, settings.recordWithAudioPlayback),
```

Comment:
Missed this on the first pass. You need to update the dependency array being passed to useMemo to ensure the audio clip changes whenever the setting changes.

Reply:
I'm not sure I understand, I added now the setting to the array below this line if that's what you meant. Correct me if i'm wrong

Reply:
Your change is the right one. If you're interested in learning why you can look up react hooks and dependency arrays.